src/cmds/get/group.ts

```diff
@@ -0,0 +1,20 @@
+import { log, logFailure } from '../../logger'
+import { client } from '../../api/client'
+
+const CMD = 'group'
+
+function addGetGroupCmd(cmd) {
+    cmd.command(CMD).argument('group', 'Contract group to get the ABI for').action(getGroup)
+}
+
+async function getGroup(group: string) {
```

Comment:
Reason we aren't validating the `group` format here like we do in multiple other commands with `isValidContractGroup`?